src/pages/City/City/CheckActiveMembersForm.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import classes from './Form.module.css'
+import {Col, Form, Select, Button, Avatar, Row, PageHeader } from 'antd';
+import { redirectCities, removeRegion } from "../../../api/regionsApi";
+import { useParams, useHistory, Link, useRouteMatch } from 'react-router-dom';
+import "./CheckActiveMembersForm.less";
+import Title from 'antd/lib/skeleton/Title';
+type CheckActiveCitiesForm = {
+    onAdd: () => void;
+}
+
+
+
+const CheckActiveMembersForm  = (props: any)=>{
+    
+    const  { onAdd } = props;
+    const [form] = Form.useForm();
+    const { id } = useParams();
+    const { url } = useRouteMatch()
+    const history = useHistory();
+
+    const [activeCities, setActiveCities] = useState<any[]>([]);
+    const handleSubmit = async ()=>{
+        onAdd();
+        }
+       
+    return <Form
+            name="basic"
+            onFinish={handleSubmit}
+            form = {form}
+            >
+         <Form.Item
+             className={classes.formField}
+         > {props.members.length !== 0 ? (
+             <React.Fragment>
+                <PageHeader 
+                title = "Необхідно відкріпити членів: "
+                className="site-page-header" 
+                />
+                        <Row className= "cityItems" justify="center">
+                        {props.members.map((member: any) => ( 
+                            <Link to={"/userpage/main/"+member.userId} target="_blank">
+                                <Col
+                                key={member.id}
+                                >
+                                <div onClick={() => history.push(`/cities/${member.id}`)}>
+                                    <Avatar size={64} src={member.user.imagePath} />
+                                    <p>{member.user.firstName}</p>
+                                </div>
+                                </Col>
+                            </Link>
+                        ))}
+                        </Row> 
+                </React.Fragment>
+         ) : null}
+            {props.admins.length !== 0 ? (
+                <React.Fragment>
+                <PageHeader 
+             title = "Необхідно відкріпити членів проводу: "
+             className="site-page-header" 
+             />
```

Comment:
add tabulation for title, classname, and closing tag, so it is on the same tab as opening tag